staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -44,6 +44,9 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// Parallelism defines the amount of parallelism that should be used in scheduler algorithms and some plugins.
+	Parallelism int32 `json:"parallelism"`
```

Comment:
Make this a pointer and add `omitempty` to the serialization

Reply:
So in `KubeSchedulerConfig` can i assign thevalue of parallelism like
```
&config.KubeSchedulerConfiguration{
		Parallelism:        func(val int32) *int32 { return &val }(10),
...
```

as it a pointer, or some other way is suggested ?